var/spack/repos/builtin/packages/kitty/package.py

```diff
@@ -49,7 +103,9 @@ class Kitty(Package):
     depends_on("fixesproto", when=sys.platform != "darwin")
     depends_on("dbus", when=sys.platform != "darwin")
     depends_on("xkeyboard-config", when=sys.platform != "darwin")
+    depends_on("librsync")
+    depends_on("lcms2")
 
     def install(self, spec, prefix):
-        with working_dir(self.build_directory):
-            self.python("-s", "setup.py", "linux-package", "--prefix={0}".format(prefix))
+        with working_dir("."):
```

Comment:
You can actually remove this line if you don't need to change directories

Reply:
Ah! so you mean we don't need `with working_dir(...)` anymore?

Reply:
Yep